src/plugins/base/auth/components/EmailVerif.js

```diff
@@ -15,42 +15,59 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with RACTF.  If not, see <https://www.gnu.org/licenses/>.
 
-import React, { useState, useEffect } from "react";
+import React, { useState } from "react";
 import { Redirect } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { useTranslation } from "react-i18next";
 
-import { Spinner, Form } from "@ractf/ui-kit";
+import { Button, Form, Spinner, Markdown } from "@ractf/ui-kit";
```

Comment:
It would be nice for these to be alphabetically ordered - perhaps another PR is needed to make the linter check for this.

Reply:
Duly noted